pkgs/top-level/all-packages.nix

```diff
@@ -29698,6 +29698,10 @@ with pkgs;
 
   cutecapture = callPackage ../applications/video/cutecapture { };
 
+  cwtch-ui = callPackage ../by-name/cw/cwtch-ui/package.nix {
+    flutter = flutter313;
```

Comment:
I feel a bit iffy about calling flutter319 here. flutter319 should also be available in by-name packages. 
In other packages I see patter like this to pin version.
https://github.com/NixOS/nixpkgs/blob/5bbede523a48ac140ee25ea6e203f86d79f415b6/pkgs/by-name/bi/bitwarden-desktop/package.nix#L29

```
{ flutter313, ...}: let
   flutter = flutter313;
in
```

Reply:
I was following the [recommendations in the by-name docs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#changing-implicit-attribute-defaults). That argues that the benefit of doing it this way is that it permits someone else to `cwtch-ui.override { flutter = ... }` if they want to.

Reply:
As I predicted, cwtch 1.15.0, released this week, updates to support the latest flutter, so this entry in `all-packages.nix` has been removed.